jupyter/nmm-gui.ipynb

```diff
@@ -822,7 +980,14 @@
    "metadata": {},
    "source": [
     "### poundMillInGui\n",
-    "(TODO: Erklärung)"
+    "\n",
+    "Die Funktion `poundMillInGui` entfernt einen gegnerischen Spielerstein und beendet somit einen Mühlenzug. Die Funktion hat dabei ein Argument:\n",
+    "\n",
+    "- `stone` ist die Koordinate des gegnerischen Spielersteins, der entfernt werden soll.\n",
+    "\n",
+    "Ob ein Spielerstein entfernt werden kann, wird mit der Funktion `validateNewState` validiert.\n",
+    "\n",
+    "In der Spielerphase 1 kann es aber vorkommen, dass ein Spieler zwei Mühlen ausführen kann. In diesem Fall kann das nicht von der Funktion `validateNewState` ausführt werden, weil der Zug noch nicht abgeschlossen ist und somit nicht in der Menge `nextStates` auftritt. In diesem Fall muss die Validierung von der Funktion selber ausführt werden."
```

Comment:
Die Phasen haben auch in deutschen Namen, wir sollten die Wahrscheinlich am besten verwenden.

Das `aber` im ersten Satz ist unnötig.

Und heißt es nicht Mühlen schlagen anstatt `ausführen`?